modules/bibcirculation/lib/bibcirculation_dblayer.py

```diff
@@ -591,7 +591,46 @@ def get_pdf_request_data(status):
 ###
 
 
+def calculate(s, e, n):
+    """
+    Get details about a book is on loan period
+    during time interval
+
+    @param s: start date.
+    @type s: string
+
+    @param e: end date.
+    @type e: string
+
+    @param n: number of top results.
+    @type n: string
+
+    @return list with Record No, Loan Period,
+    Item copies, Volume and Item List.
+    """
+    s = str(s)
+    e = str(e)
```

Comment:
do you need this two here? If you say they are strings in the doc string, ti should be enough I guess